Empty pairings.json file causes app to not start after pairing two devices

# Bug Report

## Description

After doing a successful pairing and closing the app, it can no longer be started again, because the `pairings.json` file is empty.

Logs:
```
INF 2023-10-04 11:01:42.919-04:00 starting application controller...         topics="status-app" tid=246112 file=nim_status_client.nim:193
DBG 2023-10-04 11:01:42.919-04:00 init keycard using                         topics="keycard-service" tid=246112 file=service.nim:84 pairingsJson=/home/jonathan/dev/status-desktop/Status/data/keycard/pairings.json
DBG 2023-10-04 11:01:42.919-04:00 initialization response:                   topics="keycard-service" tid=246112 file=service.nim:87 initResp="unexpected end of JSON input"
Traceback (most recent call last)
/home/jonathan/dev/status-desktop/src/nim_status_client.nim(200) nim_status_client
/home/jonathan/dev/status-desktop/src/nim_status_client.nim(194) mainProc
/home/jonathan/dev/status-desktop/src/app/boot/app_controller.nim(402) start
/home/jonathan/dev/status-desktop/src/app/modules/startup/io_interface.nim(22) load
/home/jonathan/dev/status-desktop/src/app/modules/startup/module.nim(142) load
/home/jonathan/dev/status-desktop/src/app/modules/startup/io_interface.nim(112) setSelectedLoginAccount
/home/jonathan/dev/status-desktop/src/app/modules/startup/module.nim(300) setSelectedLoginAccount
/home/jonathan/dev/status-desktop/src/app/modules/startup/controller.nim(536) cancelCurrentFlow
/home/jonathan/dev/status-desktop/src/app_service/service/keycard/service.nim(147) cancelCurrentFlow
/home/jonathan/dev/status-desktop/vendor/nim-keycard-go/keycard_go.nim(21) keycardCancelFlow
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
make: *** [Makefile:794: run-linux] Segmentation fault (core dumped)
```

Indeed, the file contains nothing.

## Steps to reproduce

0. Have an account ready to pair
1. Open a new status instance
2. Pair the two using the sync code
3. All goes well
4. Close the second app 
5. Try to reopen it

It doesn't open with the error above

#### Expected behavior

App opens normally, ready to login

#### Actual behavior

App doesn't start

### Additional Information

- Status desktop version: master (`c6449f67c622a824a95c9c58ed6a091181963820
- Operating System: Ubuntu 20

--- a/services/wallet/keycard_pairings.go
+++ b/services/wallet/keycard_pairings.go
@@ -28,6 +28,10 @@ func (kp *KeycardPairings) GetPairingsJSONFileContent() ([]byte, error) {
 }
 
 func (kp *KeycardPairings) SetPairingsJSONFileContent(content []byte) error {
+	if len(content) == 0 {
+		// Nothing to write
+		return nil
+	}
 	_, err := os.Stat(kp.pairingsFile)
 	if os.IsNotExist(err) {
 		dir, _ := filepath.Split(kp.pairingsFile)

